ipn/ipnlocal/local.go

```diff
@@ -3171,6 +3171,12 @@ func (b *LocalBackend) peerAPIServicesLocked() (ret []tailcfg.Service) {
 			Port:  1, // version
 		})
 	}
+	if b.appConnector != nil {
+		ret = append(ret, tailcfg.Service{
+			Proto: tailcfg.AppConnector,
+			Port:  1, // version
```

Comment:
Add this to the docstring on `Service.Proto` in the existing style

Reply:
done, though I think they should move near to the constants to scale better over time